packages/nodes-base/nodes/Microsoft/OneDrive/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const MICROSOFT_DRIVE_FILE_URL_REGEX =
+	'https:\/\/onedrive.live.com(?:\/.*?(\&|\?)(id=|resid=))(.+)(?:\&.*)';
```

Comment:
This regex is invalid and doesn't work. When trying use `Watch A Selected File` with RLC URL the input field is also vanishing due to this.